react/react1/week3/react1-week3/src/components/TodoListRow.jsx

```diff
@@ -0,0 +1,38 @@
+import {useState} from "react"
+import Border from "../Border"
+
+const TodoListRow =(props)=>{
+    const { todo, deleteItem, id } = props
+    const [checked, setChecked]=useState(false)
+    const [editing, setEditing]=useState(false) 
+    const [descriptionInputValue, setDescriptionInputValue] = useState(todo.description)
+
+
+    const updateEditButton =() =>{
+        if (editing === true ){
+            console.log(descriptionInputValue)
+            props.updateDescription(id, descriptionInputValue)
+        }
+        setEditing((currentValue)=>!currentValue)
+
+    }
+
+    return(
+        < Border>
```

Comment:
I didn't understand this, can you help me?

Reply:
Border is a component that wraps another component in a black border using react children (I put the styling in App.css)